Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.yml

```diff
@@ -0,0 +1,2770 @@
+          The timestamp is in UTC.
+        type: Date
+      - contextPath: Cofense.IOC.metadata.quarantine.first_quarantined_at
+        description: Date and time cofense vision quarantined the first email due to
+          this IOC. The timestamp is in UTC.
+        type: Date
+      - contextPath: Cofense.IOC.metadata.quarantine.last_quarantined_at
+        description: Date and time cofense vision quarantined the last email due to
+          this IOC. The timestamp is in UTC.
+        type: Date
+      - contextPath: Cofense.IOC.metadata.quarantine.match_count
+        description: Number of unique emails that matched the IOC while the IOC was
+          active.
+        type: Number
+      - contextPath: Cofense.IOC.metadata.quarantine.quarantine_count
+        description: Number of recipients who received emails matching the IOC while
+          the IOC was active.
+        type: Number
+      - contextPath: Cofense.IOC.metadata.quarantine.expired
+        description: Whether the IOC is expired (true) or not expired (false).
+        type: Boolean
+      - contextPath: Cofense.IOC.metadata.quarantine.wildcard
+        type: Boolean
+        description: Whether the wildcard matching is set (true) or not (false).
+      - contextPath: DBotScore.Indicator
+        type: String
+        description: The indicator that was tested.
+      - contextPath: DBotScore.Type
+        type: String
+        description: The indicator type.
+      - contextPath: DBotScore.Vendor
+        type: String
+        description: The vendor used to calculate the score.
+      - contextPath: DBotScore.Score
+        type: Number
+        description: The actual score.
+      - contextPath: Domain.Name
+        description: The domain name.
+        type: String
+      - contextPath: Domain.Malicious.Description
+        description: A description of the malicious domain.
+        type: String
+      - contextPath: Domain.Malicious.Vendor
+        description: The vendor who reported the domain as malicious.
+        type: String
+      - contextPath: URL.Data
+        description: The URL.
+        type: String
+      - contextPath: URL.Malicious.Description
+        description: A description of the malicious URL.
+        type: String
+      - contextPath: URL.Malicious.Vendor
+        description: The vendor who reported the URL as malicious.
+        type: String
+      - contextPath: Email.Address
+        description: The sender of the email.
+        type: String
+      - contextPath: Email.Malicious.Description
+        description: A description of the malicious email.
+        type: String
+      - contextPath: Email.Malicious.Vendor
+        description: The vendor who reported the email as malicious.
+        type: String
+      - contextPath: File.SHA256
+        description: The SHA256 hash of the file.
+        type: String
+      - contextPath: File.MD5
+        description: The MD5 hash of the file.
```

Comment:
regarding these fields, are they available on Cofense, and can be mapped in the IP indicator? Same for the rest of the standard indicator objects.

Reply:
Yes, we've mapped some of the standard fields to common indicators fields.